applications/OptimizationApplication/tests/shell-thickness-opt-test-new/optimization_parameters.json

```
@@ -0,0 +1,130 @@
+{
+    "problem_data": {
+        "parallel_type": "OpenMP",
+        "echo_level": 0
+    },
+    "model_parts": [
+        {
+            "settings": {
+                "model_part_name": "shell",
+                "domain_size": 3,
+                "input_filename": "shell"
+            }
+        }
+    ],
+    "analyses": [
+        {
+            "name": "shell_static",
+            "type": "kratos_analysis_execution_policy",
+            "settings": {
+                "model_part_names": [
+                    "shell"
+                ],
+                "analysis_module": "KratosMultiphysics.StructuralMechanicsApplication",
+                "analysis_type": "StructuralMechanicsAnalysis",
+                "analysis_settings": {
+                    "@include_json": "primal_parameters.json"
+                }
+            }
+        }
+    ],
+    "responses": [
+        {
+            "name": "mass_shell",
+            "type": "mass_response_function",
+            "settings": {
+                "evaluated_model_part_names": [
+                    "shell"
+                ]
+            }
+        },
+        {
+            "name": "strain_energy_shell",
+            "type": "linear_strain_energy_response_function",
+            "settings": {
+                "evaluated_model_part_names": [
+                    "shell"
+                ],
+                "primal_analysis_name": "shell_static"
+            }
+        }
+    ],
+    "controls": [
+        {
+            "name": "thickness_control",
+            "type": "thickness.shell_thickness_control",
+            "settings": {
+                "controlled_model_part_names": [
+                    "shell"
+                ],
+                "filter_settings": {
+                    "type": "implicit",
+                    "radius": 0.2,
+                    "linear_solver_settings": {
+                        "solver_type": "LinearSolversApplication.pardiso_lu"
+                    }
+                },
+                "initial_thickness": 0.1,
+                "fixed_model_parts": [
+                    "shell.top_edge"
+                ],
+                "fixed_model_parts_thicknesses": [
+                    0.2
+                ]
```

```suggestion
                "fixed_model_part_thicknesses": {
                    "shell.top_edge": 0.2
                },
```
I would use a dictionary here so it is clear and no index followups required.